Formula/tomsfastmath.rb

```diff
@@ -0,0 +1,64 @@
+class Tomsfastmath < Formula
+  desc "Fast large integer arithmetic library written in portable ISO C"
+  homepage "https://www.libtom.net/TomsFastMath/"
+  url "https://github.com/libtom/tomsfastmath/releases/download/v0.13.1/tfm-0.13.1.tar.xz"
+  sha256 "47c97a1ada3ccc9fcbd2a8a922d5859a84b4ba53778c84c1d509c1a955ac1738"
+
+  depends_on "libtool" => :build
+
+  # Fixes some issues in `makefile.shared`, remove on next release.
+  patch :DATA
```

Comment:
The changes in this patch were already done upstream, but a release with the changes hasn't been made yet.